nodebuilder/p2p/network.go

```diff
@@ -46,6 +46,15 @@ var networksList = map[Network]struct{}{
 	Private: {},
 }
 
+// networkAliases is a strict list of all known long-standing networks
+// mapped from the string representation of their *alias* (rather than
+// their actual value) to the Network.
+var networkAliases = map[string]Network{
+	"arabica": Arabica,
+	"mamaki":  Mamaki,
+	"private": Private,
```

Comment:
Nit optional suggestion to make `Name() string`  method for `Network` type with static value.